kinto/core/resource/viewset.py

```diff
@@ -120,13 +120,13 @@ def get_view_arguments(self, endpoint_type, resource_cls, method):
         args.update(**endpoint_args)
 
         if method.lower() in map(str.lower, self.validate_schema_for):
-            schema = PartialSchema()
+            schema = RequestSchema()
             record_schema = self.get_record_schema(resource_cls, method)
             record_schema.name = 'body'
             schema.add(record_schema)
             args['schema'] = schema
         else:
-            args['schema'] = SimpleSchema()
+            args['schema'] = RequestSchema()
```

Comment:
One was `Partial` and the other one `Simple`. Is that ok to now give the same behaviour to both?